core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -125,6 +125,21 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
         failedTaskSetReason = reason
         failedTaskSetException = exception
       }
+      override def taskEnded(
+        task: Task[_],
+        reason: TaskEndReason,
+        result: Any,
+        accumUpdates: Seq[AccumulatorV2[_, _]],
+        taskInfo: TaskInfo): Unit = {
```

Comment:
nit: 4 space indentation for method params (sorry probably my bad in the snippet I gave you)

Reply:
Done